pallets/asset-manager/src/migrations.rs

```diff
@@ -65,22 +67,18 @@ impl<T: GetStorageVersion + Config + PalletInfoAccess> OnRuntimeUpgrade
     #[cfg(feature = "try-runtime")]
     fn pre_upgrade() -> Result<(), &'static str> {
         let storage_version = <T as GetStorageVersion>::on_chain_storage_version();
-
         if storage_version >= 1 {
             return Err("Storage version is >= 1, the migration won't be executed.");
         }
-
         Ok(())
     }
 
     #[cfg(feature = "try-runtime")]
     fn post_upgrade() -> Result<(), &'static str> {
         let storage_version = <T as GetStorageVersion>::on_chain_storage_version();
-
         if storage_version < 1 {
             return Err("Storage version is >= 1, the migration won't be executed.");
         }
-
         let acala = (2000, 3); // karura has 3 asset locations on calamari.
```

Comment:
How many parachains we're supporting, it should be more than these, right?

Reply:
I think when we did this migration that was all of them. Right now we have KSM and PHA as well